coffee-chats/src/main/java/com/google/step/coffee/servlets/ChatRequestServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.step.coffee.servlets;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/chat-request")
```

Comment:
I propose we use `/api` as a prefix to all of the endpoints. Among other things, it makes routing in the dev reverse proxy easier.

Reply:
Done